src/main/java/org/aszjch/demoapp/infrastructure/storage/CustomMultipartFile.java

```diff
@@ -0,0 +1,62 @@
+package org.aszjch.demoapp.infrastructure.storage;
+
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+
+public class CustomMultipartFile implements MultipartFile {
```

Comment:
Chyba to mamy wyżej w klasie wew.

Reply:
Rzeczywiście, jakoś magicznie musiałem je zduplikować... Już wyrzucam jedno z nich